bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -122,297 +109,175 @@ If you did not manually create things in the *.things file, the Netatmo Binding
-| HealthIndex         | String               | Health index (healthy, fine, fair, poor, unhealthy)      |
-| Co2                 | Number:Dimensionless | Air quality                                              |
-| Temperature         | Number:Temperature   | Current temperature                                      |
-| TempTrend           | String               | Temperature evolution trend (up, down, stable)           |
-| Noise               | Number:Dimensionless | Current noise level                                      |
-| Pressure            | Number:Pressure      | Current pressure                                         |
-| PressTrend          | String               | Pressure evolution trend for last 12h (up, down, stable) |
-| AbsolutePressure    | Number:Pressure      | Absolute pressure                                        |
-| Humidity            | Number:Dimensionless | Current humidity                                         |
-| MinTemp             | Number:Temperature   | Minimum temperature on current day                       |
-| MaxTemp             | Number:Temperature   | Maximum temperature on current day                       |
-| DateMinTemp         | DateTime             | Date when minimum temperature was reached on current day |
-| DateMaxTemp         | DateTime             | Date when maximum temperature was reached on current day |
-| TimeStamp           | DateTime             | Timestamp when data was measured                         |
-| LastStatusStore     | DateTime             | Last status store                                        |
-| WifiStatus          | Number               | Signal strength (0 for no signal, 1 for weak, 2 for average, 3 for good or 4 for excellent) |
-| Location            | Location             | Location of the device                                   |
+| Channel Group       | Channel Id           | Item Type            | Description                                      |
+|---------------------|----------------------|----------------------|--------------------------------------------------|
+| noise               | value                | Number:Dimensionless | Current noise level                              |
+| humidity            | value                | Number:Dimensionless | Current humidity                                 |
+| humidity            | humidex              | Number               | Computed Humidex index                           |
+| humidity            | humidex-scale        | Number               | Humidex index appreciation                       |
+| humidity            | heat-index           | Number:Temperature   | Computed Heat Index                              |
+| humidity            | dewpoint             | Number:Temperature   | Computed dewpoint temperature                    |
+| humidity            | dew-point-depression | Number:Temperature   | Computed dewpoint depression                     |
```

Comment:
Can't check, but chances are that name is actually `dewpoint-depression`.

Reply:
True